tests/legacy-cli/e2e/tests/commands/serve/assets.ts

```diff
@@ -0,0 +1,23 @@
+import assert from 'assert';
```

Comment:
I am not totally following the test, it's named `assets` but it's testing only for `favicon.ico`. Which is covered by other tests. 

https://github.com/angular/angular-cli/blob/e2f92ab957e797d8085616fcdea0b73344e614af/tests/legacy-cli/e2e/tests/basic/serve.ts#L22

It seems that the tests itself, is mostly testing that the index is always server on not found pages?